firehose/parsers/flawfinder.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+#
+#   Copyright 2017 David Carlos  <ddavidcarlos1392@gmail.com>
+#   This library is free software; you can redistribute it and/or
+#   modify it under the terms of the GNU Lesser General Public
+#   License as published by the Free Software Foundation; either
+#   version 2.1 of the License, or (at your option) any later version.
+#
+#   This library is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#   Lesser General Public License for more details.
+#
+#   You should have received a copy of the GNU Lesser General Public
+#   License along with this library; if not, write to the Free Software
+#   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
+#   USA
+
+import sys
+import re
+from subprocess import check_output
+from firehose.model import Message, Point, \
+    File, Location, Generator, Metadata, Analysis, Issue
+
+
+def main():
+    """ Main entry to flawfinder parser """
+    try:
+        arg_file = sys.argv[1]
+        report = open(arg_file, 'r')
+    except IOError:
+        print("File note found")
```

Comment:
Typo: "note" should be "not".
But an IOError is not necessarily "file not found"; it could be a permissions error, or an attempt to open a directory rather than a file, etc.   Maybe just lose this clause?